offchain-modules/lib/src/monitor/account_monitor.rs

```diff
@@ -0,0 +1,125 @@
+use crate::util::ckb_tx_generator::Generator;
+use crate::util::ckb_util::{get_secret_key, parse_privkey};
+use crate::util::config::ForceConfig;
+use crate::util::eth_util::{secret_key_address, Web3Client};
+use anyhow::{anyhow, Result};
+use ckb_types::packed::Script;
+use ethereum_types::U256;
+use force_sdk::cell_collector::get_all_live_cells_by_lockscript;
+use secp256k1::SecretKey;
+use std::ops::Div;
+use web3::types::Address;
+
+pub struct AccountMonitor {
+    web3_client: Web3Client,
+    generator: Generator,
+    eth_addresses: Vec<Address>,
+    ckb_lockscripts: Vec<Script>,
+    alarm_url: String,
+    ckb_alarm_balance: u64,
+    eth_alarm_balance: u64,
+    eth_balance_conservator: String,
+    ckb_balance_conservator: String,
+}
+
+impl AccountMonitor {
+    pub async fn new(
+        config_path: String,
+        network: Option<String>,
+        alarm_url: String,
+        ckb_alarm_balance: u64,
+        eth_alarm_balance: u64,
+        eth_balance_conservator: String,
+        ckb_balance_conservator: String,
+    ) -> Result<Self> {
+        let force_config = ForceConfig::new(&config_path)?;
+        let eth_rpc_url = force_config.get_ethereum_rpc_url(&network)?;
+        let ckb_rpc_url = force_config.get_ckb_rpc_url(&network)?;
+        let ckb_indexer_url = force_config.get_ckb_indexer_url(&network)?;
+        let ckb_privkeys = force_config.get_ckb_private_keys(&network)?;
+        let eth_privkeys: Vec<String> = force_config.get_ethereum_private_keys(&network)?;
+
+        let mut eth_addresses: Vec<Address> = vec![];
+        for eth_privkey in eth_privkeys.into_iter() {
+            let eth_private_key =
+                ethereum_types::H256::from_slice(hex::decode(eth_privkey)?.as_slice());
+            let eth_key = SecretKey::from_slice(&eth_private_key.0)?;
+            let from = secret_key_address(&eth_key);
+            eth_addresses.push(from);
+        }
+        let mut ckb_lockscripts: Vec<Script> = vec![];
+        for ckb_privkey in ckb_privkeys.into_iter() {
+            let privkey = get_secret_key(&ckb_privkey)?;
+            let lockscript = parse_privkey(&privkey);
+            ckb_lockscripts.push(lockscript);
+        }
+        Ok(AccountMonitor {
+            web3_client: Web3Client::new(eth_rpc_url),
+            generator: Generator::new(ckb_rpc_url, ckb_indexer_url, Default::default())
+                .map_err(|e| anyhow!("failed to crate generator: {}", e))?,
+            eth_addresses,
+            ckb_lockscripts,
+            alarm_url,
+            ckb_alarm_balance,
+            eth_alarm_balance,
+            eth_balance_conservator,
+            ckb_balance_conservator,
+        })
+    }
+
+    pub async fn start(&mut self) -> Result<()> {
+        let mut msg = " ".to_string();
+        let eth_decimal: U256 = U256::from(10u128.pow(18));
+        for eth_address in self.eth_addresses.iter() {
```

Comment:
better put the account index before message, eg. "eth account[0] 0x123456666..666 balance: 0 eth"